app/javascript/dashboard/api/ApiClient.js

```diff
@@ -17,7 +17,10 @@ class ApiClient {
   }
 
   create(data) {
-    return axios.post(this.url, data);
+    return axios.post(this.url, data)
```

Comment:
Why was this changed?

Reply:
I was thinking I needed this to bubble the error up.

I was mistaken, it is not required. I will remove this change.